src/index.js

```diff
@@ -53,7 +53,9 @@ const createHelloBar = () => {
   div.appendChild(p);
   div.appendChild(btn);
   a.appendChild(div);
-  header.insertBefore(a, header.firstChild);
+  if (header) header.insertBefore(a, header.firstChild);
```

Comment:
Since you added this logic, you can add a test for it: _"createHelloBar should not fail when there is no a header"_